pkg/virt-operator/resource/generate/components/validatingadmissionpolicy.go

```diff
@@ -147,8 +148,21 @@ func NewHandlerV1ValidatingAdmissionPolicy(virtHandlerServiceAccount string) *ad
 					Name:       "newAnnotations",
 					Expression: "object.metadata.annotations",
 				},
+				{
+					Name:       "requestMatchNode",
+					Expression: "object.metadata.name == request.userInfo.extra['authentication.kubernetes.io/node-name'][0]",
+				},
+				{
+					Name:       "hasNode",
```

Comment:
Any chance we could unit-test this? Is it even possible for the annotation to be missing?

Reply:
For unit test, yes afaik possible, do you want me to add it?

Reply:
@jean-edouard Can I get away without the unit test for now? It adds a ton of dependencies and I will need to sort Bazel first.